airbyte-webapp/src/views/Connector/ConnectorDocumentationLayout/ConnectorDocumentationLayout.tsx

```diff
@@ -0,0 +1,45 @@
+import { faGripLinesVertical } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React from "react";
+import { ReflexContainer, ReflexElement, ReflexSplitter } from "react-reflex";
+import styled from "styled-components";
+
+import { useDocumentationPanelContext } from "./ConnectorDocumentationContext";
+
+const PanelGrabber = styled.div`
+  height: 100vh;
+  padding: 6px;
+  display: flex;
+`;
+
+const GrabberHandle = styled(FontAwesomeIcon)`
+  margin: auto;
+  height: 25px;
+  color: ${({ theme }) => theme.greyColor20};
+`;
+
+//NOTE: ReflexElement will not load its contents if wrapped in an empty jsx tag along with ReflexSplitter.  They must be evaluated/rendered separately.
+
+export const ConnectorDocumentationLayout = ({ children }: { children: [React.ReactNode, React.ReactNode] }) => {
+  const [left, right] = children;
+
+  const { documentationPanelOpen } = useDocumentationPanelContext();
+
+  return (
+    <ReflexContainer orientation="vertical" windowResizeAware={true}>
```

Comment:
I noticed this a few places, but the general convention is to not include a value:
```suggestion
    <ReflexContainer orientation="vertical" windowResizeAware>
```

We could also enable this rule to auto fix it:
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md